packages/front-end/components/Settings/NewDataSourceForm.tsx

```diff
@@ -540,6 +595,9 @@ const NewDataSourceForm: FC<{
             />
           </div>
         )}
+        {selectedSchema?.value !== "custom" && datasource.type ? (
+          <div className="alert alert-info"></div>
```

Comment:
Is this a mistake?  It causes an extra empty blue alert
![image](https://github.com/growthbook/growthbook/assets/1087514/67e0e55c-c8ef-452c-9f6f-db9880d3426e)

Reply:
Yea - that's an artifact from when I was exploring adding an additional exit hatch right here. Removed the offending code.